From 0bf2e53efea145e9ec770ee64722488953d4d7b1 Mon Sep 17 00:00:00 2001
From: Maciej Wereski <m.wereski@partner.samsung.com>
Date: Fri, 10 Oct 2014 15:08:47 +0200
Subject: [PATCH 14/24] Set DBUS_SESSION_BUS_ADDRESS for user instance

This was at first set by user-session@.service, then by many services
(bt-fwrk) and scripts (weston.sh). This patch makes it again available
for whole user environment.

Signed-off-by: Maciej Wereski <m.wereski@partner.samsung.com>
---
 units/user@.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/user@.service.in b/units/user@.service.in
index 08b1cd7..d21f761 100644
--- a/units/user@.service.in
+++ b/units/user@.service.in
@@ -17,3 +17,4 @@ ExecStart=-@rootlibexecdir@/systemd --user
 SmackExecLabel=User
 Slice=user-%i.slice
 KillMode=mixed
+Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/%U/dbus/user_bus_socket
-- 
1.8.4.5

